Fix compatibility with Passport 0.3.0

voxpelli opened the pull request:
One has to depend on the `passport-strategy` package rather than the main package now to avoid conflicts between Passport versions (errors like: `TypeError: Cannot set property 'user' of undefined`). This PR fixes that.

(This PR also+ ensures that the exposed version matches the one in `package.json` + adds a `.gitignore` for `/node_modules` – I can remove these extra changes from the PR if you want)

After this has been merged I would suggest publishing a `1.0.0` release of this module. It works well for what it is supposed to do, I've used it in the dev environments of many projects of mine to avoid Twitter/GitHub login locally, so no need for a sub-`1.0.0` version as I see it.

If you want help, feel free to add me to the module here and on NPM and I can merge and publish this fix myself if you perhaps no longer have an active interest in the module.

voxpelli committed:
Expose version from package.json

voxpelli committed:
Use the passport-strategy module for 0.3.0 compat

voxpelli committed:
Added gitignore

voxpelli commented:
Ping, anyone wants to take a look at this?

olafveerman commented:
@voxpelli We are not actively maintaining this package anymore, but would be happy to transfer ownership  to you or another org if you'd be interested in doing so.

olafveerman commented:
Added you as an admin to the repo. Feel free to transfer it to the org that will maintain this from now on.

voxpelli commented:
Thanks @olafveerman! Did you add me to the npm module as well?

olafveerman commented:
@voxpelli Unfortunately we don't have access to the npm package

voxpelli commented:
Fixed in fork. Updated README here to point to fork: https://github.com/developmentseed/passport-dummy/commit/38f4739a588cc5da062e471400808d49ac60e1f1 As a package can't be published to the original package name a fork is unavoidable. Closing this.
